.travis.yml

```diff
@@ -327,11 +384,12 @@ matrix:
             - ubuntu-toolchain-r-test
 
     - os: linux
-      env: TOOLSET=gcc COMPILER=g++-6 CXXSTD=c++03 TEST_SUITE="compile_fail examples"
+      env: TOOLSET=gcc COMPILER=g++-6 CXXSTD=c++03 TEST_SUITE="examples compile_fail"
```

Comment:
Just curious, why the gratuitous change in order?

Reply:
Nothing special. Could be reverted. Should be no difference. Maybe, that was accidentally changed while crafting.